mapfishapp/src/main/filtered-resources/WEB-INF/jsp/index.jsp

```diff
@@ -4,19 +4,50 @@
 <%@ page import="java.util.*" %>
 <%@ page import="org.georchestra.mapfishapp.ws.Utf8ResourceBundle" %>
 <%@ page import="org.apache.commons.lang.StringUtils" %>
+<%@ page import="org.springframework.web.context.support.WebApplicationContextUtils" %>
+<%@ page import="org.springframework.context.ApplicationContext" %>
+<%@ page import="org.springframework.web.servlet.support.RequestContextUtils" %>
+<%@ page import="org.georchestra.commons.configuration.GeorchestraConfiguration" %>
+
+
 <%@ page contentType="text/html; charset=UTF-8" %>
 <%@ page pageEncoding="UTF-8" %>
 <%@ page isELIgnored="false" %>
 <%
 Boolean anonymous = true;
 Boolean admin = false;
+Boolean georDatadirActivated = false;
+
+String instanceName = null;
+String defaultLanguage = null;
+String georCustomPath = "/app/js/GEOR_custom.js";
+
+try {
+  ApplicationContext ctx = RequestContextUtils.getWebApplicationContext(request);
+  instanceName = ctx.getBean(GeorchestraConfiguration.class).getProperty("instance");
+  defaultLanguage = ctx.getBean(GeorchestraConfiguration.class).getProperty("language");
+  if ((ctx.getBean(GeorchestraConfiguration.class) != null)
+    && (((GeorchestraConfiguration) ctx.getBean(GeorchestraConfiguration.class)).activated())) {
+      georDatadirActivated = true;
+      georCustomPath = "/ws/app/js/GEOR_custom.js";
+    }
+} catch (Exception e) {
+}
 
 // the context path (might not be the public context path ! -> to be improved with https://github.com/georchestra/georchestra/issues/227)
 String context = request.getContextPath().split("-")[0]; // eg /mapfishapp
 
 String lang = request.getParameter("lang");
 if (lang == null || (!lang.equals("en") && !lang.equals("es") && !lang.equals("ru") && !lang.equals("fr") && !lang.equals("de"))) {
+  if (defaultLanguage != null) {
+    lang = defaultLanguage;
+  }
+  else {
     lang = "${language}";
+  }
+}
```

Comment:
I'm not sure this logic enables the user to override the defaultLanguage by passing ?lang=XX to the webapp